src/User.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Icepace;
+
+class User
+{
+    protected int $id;
+    protected string $username;
+    protected string $bio;
+    protected string $avatar;
+    protected string $hashed_pass;
+
+    public function __construct(int $id = 0, string $username = '', string $hashed_pass = '', string $bio = '', string $avatar = '')
```

Comment:
please add a doc block comment to this constructor explaining that it is to be used for unit testing ONLY. the only reason this constructor exists is for unit testing, so we want to warn developers not to use it otherwise. that way, because doc blocks are magical, if someone tries to call the method it will come up with the comment as a tooltip.

Reply:
added